diff --git a/libtiledbvcf/cmake/Modules/FindHTSlib.cmake b/libtiledbvcf/cmake/Modules/FindHTSlib.cmake
index 3e5b4e7..3688982 100644
--- a/libtiledbvcf/cmake/Modules/FindHTSlib.cmake
+++ b/libtiledbvcf/cmake/Modules/FindHTSlib.cmake
@@ -91,7 +91,7 @@ if (NOT HTSLIB_FOUND)
         COMMAND
           ${AUTORECONF} -i
         COMMAND
-          ./configure --prefix=${EP_INSTALL_PREFIX} LDFLAGS=${EXTRA_LDFLAGS} CFLAGS=${CFLAGS}
+          ./configure --disable-libcurl --disable-gcs --disable-aws --prefix=${EP_INSTALL_PREFIX} LDFLAGS=${EXTRA_LDFLAGS} CFLAGS=${CFLAGS}
       BUILD_COMMAND
         $(MAKE)
       INSTALL_COMMAND
